ajax/libs/jquery.touch/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "jquery.touch",
+  "filename": "jquery.touch.min.js",
+  "description": "Adds touch gesture events to jQuery",
+  "keywords": [
+    "jquery",
+    "touch"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ajlkn/jquery.touch"
+  },
+  "license": "MIT",
+  "author": "aj <@ajlkn> (https://aj.lkn.io/)",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ajlkn/jquery.touch.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.touch*"
```

Comment:
Or maybe just `jquery.touch*` next time since there is no other file types need to be excluded here.